test/fixtures/confidential_simple/main.tf

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
The fixtures should match the example path, so because we moved the example to `examples/instance_template/confidential_computing` we also need to move the fixture from `fixture/confidential_computing` to `fixture/instance_template/confidential_computing`

Reply:
As discussed offline, I changed the folder name only.